LICENSE

```diff
@@ -225,11 +225,13 @@ The following files include code modified from Michael Burman's gorilla-tsc proj
 ./tsfile/src/main/java/org/apache/iotdb/tsfile/encoding/encoder/LongGorillaEncoder.java
 ./tsfile/src/main/java/org/apache/iotdb/tsfile/encoding/encoder/SinglePrecisionEncoderV2.java
 ./tsfile/src/main/java/org/apache/iotdb/tsfile/encoding/encoder/DoublePrecisionEncoderV2.java
+./tsfile/src/main/java/org/apache/iotdb/tsfile/encoding/encoder/ZigzagEncoder.java
```

Comment:
Hi, does ZigzagEncoder also include code modified from Michael Burman's gorilla-tsc project? 
If not, we don't need to list the file here.

Reply:
Sorry for the late reply. I didn't notice the comment. The [Zigzag](https://gist.github.com/mfuerstenau/ba870a29e16536fdbaba) algorithm has a different idea from the gorilla. The benefit of Zigzag is converting negative numbers to positive ones and compressing them.